internal/rest/resources/certificates.go

```diff
@@ -72,19 +72,19 @@ func clusterCertificatesPut(s *state.State, r *http.Request) response.Response {
 			return response.BadRequest(fmt.Errorf("CA must be base64 encoded PEM key"))
 		}
 
-		err = os.WriteFile(filepath.Join(s.OS.StateDir, "cluster.ca"), []byte(req.CA), 0650)
+		err = os.WriteFile(filepath.Join(s.OS.StateDir, "cluster.ca"), []byte(req.CA), 0664)
```

Comment:
Why does microcluster need to handle CA certificates?

Reply:
I can only speculate why this was added back in time. It got added in with https://github.com/canonical/microcluster/commit/561729a0441477dc4d0b7e8f718d16b452348404.

@masnax any ideas?

Reply:
I believe it was Sunbeam that wanted this.